IDE/QNX/README.md

```diff
@@ -0,0 +1,147 @@
+# wolfTPM with QNX
+
+Instructions for creating a QNX Momentics wolfTPM Project...
+
+## Create a new QNX Application
+
+1) Create folder for libraries (lib) and includes (inc)
+2) Add library sources into "lib" directory as "wolfssl" and "wolftpm"
+3) Edit Makefile to build sources and include directories.
+
+```
+# wolfSSL and wolfTPM library includes/sources
+INCLUDES += -I./inc -I./lib/wolftpm -I./lib/wolfssl
+CCFLAGS_all += -DWOLFSSL_USER_SETTINGS -DWOLFTPM_USER_SETTINGS
+
+SRCS += $(call wildcard, lib/wolfssl/wolfcrypt/src/*.c)
+SRCS += $(call wildcard, lib/wolfssl/wolfcrypt/src/port/arm/*.c)
+SRCS += $(call wildcard, lib/wolfssl/wolfcrypt/src/port/xilinx/*.c)
+SRCS += $(call wildcard, lib/wolftpm/src/*.c)
+
+# The QNX SPI Driver
+LIBS += -lspi-master
+```
+
+4) Create `inc/user_settings.h` for all wolf specific settings:
+
+Here is a template:
+
+```
```

Comment:
nit: as a consumer of this I'd prefer the templates and diffs as files that I could copy and patches I could apply directly.

Reply:
Sounds nice, but there isn't a specific location for this user supplied file.